liblocation/src/main/java/com/mapbox/android/core/location/LocationEngineResult.java

```diff
@@ -51,8 +53,9 @@ public static LocationEngineResult create(Location location) {
    * @since 1.0.0
    */
   public static LocationEngineResult create(List<Location> locations) {
-    checkNotNull(locations, "locations can't be null");
-    return new LocationEngineResult(locations);
+    return locations != null
```

Comment:
Should we filter out `null` locations from `locations` list? Here we're only checking if the list is `null` 🤔